webapp/templates/public/menu.html.twig

```diff
@@ -10,10 +10,16 @@
 
     <div class="collapse navbar-collapse" id="menuDefault">
         <ul class="navbar-nav me-auto">
+            <li class="nav-item">
+                <a class="nav-link{% if current_route in ['public_blog_list', 'public_blog_post'] %} active{% endif %}"
+                   href="{{ path('public_blog_list') }}"><i class="fa-regular fa-newspaper"></i> Blog</a>
+            </li>
+
             <li class="nav-item">
                 <a class="nav-link{% if current_route in ['public_scoreboard', 'public_team'] %} active{% endif %}"
                     href="{{ path('public_scoreboard') }}"><i class="fas fa-list-ol"></i> Scoreboard</a>
             </li>
+
```

Comment:
vim, ze z hlediska formatovani je to asi hezci, ale potencialne by to do budoucna mohlo vyvolat konflikt z upstremem.
Idealne delat zmeny pouze v mistech, ktere se tykaji nasich zmen.